projects/mathjax-lib/README.md

```diff
@@ -1,24 +1,144 @@
-# MathjaxLib
+# mathjax-angular 
+This plugins implements the browser version of [Mathjax v3][1] into [Angular][2]
+## Feature
 
-This library was generated with [Angular CLI](https://github.com/angular/angular-cli) version 12.1.1.
+- Dynamically load MathJax library to your web application.
+- Simple typesetting using Angular directive.
+- Dynamic typesetting using expressions.
 
-## Code scaffolding
+## Install
 
-Run `ng generate component component-name --project mathjax-lib` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module --project mathjax-lib`.
-> Note: Don't forget to add `--project mathjax-lib` or else it will be added to the default project in your `angular.json` file. 
+NPM:
+```
+npm install mathjax-angular
+```
 
-## Build
+Yarn:
+```
+yarn add mathjax-angular
+```
 
-Run `ng build mathjax-lib` to build the project. The build artifacts will be stored in the `dist/` directory.
+## Configure the module
 
-## Publishing
+Load the module in the `@NgModule` class of the application. You need
+to pass a `ModuleConfiguration` instance to the `config` method to
+configure the module.
 
-After building your library with `ng build mathjax-lib`, go to the dist folder `cd dist/mathjax-lib` and run `npm publish`.
+### Example
 
-## Running unit tests
+We provide a premade exaple with the library itself, You can see the example [here][3], or alternatively run it locally by cloning the [repository][4] and running:
+```
+npm install&&npm run dev
```

Comment:
change this to two steps
1) npm install - this is for first time only to make sure all the packages are to the latest version
2)npm run dev - will launch the application in development mode and opens in browser
or
2) npm run dev production - will launch application in production mode in your default browser